tests/e2e/main.go

```diff
@@ -298,6 +325,42 @@ var rootCmd = &cobra.Command{
 			return err
 		}
 
+		// create aws creds secret
+		secretData, err = ioutil.ReadFile(filepath.Join(assetsPath, "manifests/aws-credentials.yaml"))
+		if err != nil {
+			glog.Fatalf("Error reading %#v", err)
+		}
+
+		secretObj, _, err = decode([]byte(secretData), nil, nil)
+		if err != nil {
+			glog.Fatalf("Error decoding %#v", err)
+		}
+		secret = secretObj.(*apiv1.Secret)
+		if err := createSecret(testConfig, secret); err != nil {
+			return err
+		}
+
+		// create ign config secret
+		secretData, err = ioutil.ReadFile(filepath.Join(assetsPath, "manifests/ign-config.yaml"))
```

Comment:
```go
if secret, err := readSecretFromFile(filepath.Join(assetsPath, "manifests/ign-config.yaml")); err == nil {
 	if err := createSecret(testConfig, secret); err != nil {
 		return err
 	}
} else {
 	glog.Fatalf(err)
}
```

Reply:
Prefer to fail fast. Handle the `glog.Fatalf()` first, which means putting the `createSecret()` in the `else`.